### PR TITLE
chore: change the default minimum gas fee to something non-zero

### PR DESCRIPTION
**Pull request by evan-forbes**
## Overview

update the min fee to something non zero

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by evan-forbes** [comment]
ngl, I only opened this cause I wanted to get to 1000 :tada:

File: cmd/celestia-appd/cmd/root.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ func initAppConfig() (string, interface{}) {
 	// snapshots to nodes that state sync
 	srvCfg.StateSync.SnapshotInterval = 1500
 	srvCfg.StateSync.SnapshotKeepRecent = 2
-	srvCfg.MinGasPrices = fmt.Sprintf("0%s", app.BondDenom)
+	srvCfg.MinGasPrices = fmt.Sprintf("0.001%s", app.BondDenom)
```
Comment (rootulp) [inline cmd/celestia-appd/cmd/root.go:111]: today I learned one can have fractions of 1 utia

Ref: 
- https://github.com/celestiaorg/celestia-app/blob/8b9c4c9d13fe0ccb6ea936cc26dee3f52b6f6129/testutil/network/network.go#L91
- https://github.com/celestiaorg/celestia-app/blob/8b9c4c9d13fe0ccb6ea936cc26dee3f52b6f6129/app/app.go#L95

**Review by rootulp — APPROVED** [review]

**Comment by rootulp** [comment]
Closes https://github.com/celestiaorg/celestia-app/issues/999

File: cmd/celestia-appd/cmd/root.go (hunk 1 above)
Comment (rahulghangas) [reply cmd/celestia-appd/cmd/root.go:111]: same!

**Review by SweeXordious — APPROVED** [review]

**Pull request merged**